src/components/Project/ProjectToolsDrawer/AddToProjectBalanceForm.tsx

```diff
@@ -36,8 +37,17 @@ export function AddToProjectBalanceForm({
         onDone: () => {
           setLoadingAddToBalance(false)
         },
+        onError: e => {
+          setLoadingAddToBalance(false)
+          addToBalanceForm.resetFields()
+          emitErrorNotification(e.message)
+        },
       },
     )
+
+    if (!result) {
+      setLoadingAddToBalance(false)
```

Comment:
**question**: I think this is also an error case right? Should we emit a generic error noti here?

Reply:
![CleanShot 2022-09-16 at 12 26 59](https://user-images.githubusercontent.com/2502947/190686060-c36009aa-3661-470b-aa92-885d6cc859c5.gif)

Reply:
I don't think so. The emit error is getting called in the `onError` call.